bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -131,10 +146,10 @@
         </a>
       </li>
 
-      <li class="product" id="product-marketplace">
-        <a rel="external" href="https://marketplace.firefox.com/" data-product="Firefox Marketplace">
-          <h2>{{ _('Firefox Marketplace') }}</h2>
-          <p>{{ _('Discover and download Web apps.') }}</p>
+      <li class="product" id="product-private-browsing">
+        <a href="#TODO" data-product="Private Browsing">
+          <h2>{{ _('Private Browsing') }}</h2>
+          <p>{{ _('Now with Tracking Protection.') }}</p>
```

Comment:
Ah, wait. This is not just for iOS.

The first line is available from main.lang, but this blurb should be behind another tag (yay).